roles/telemetry_logging/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- when: journal_list is defined
+  name: "Verify journal entries - {{ journal_polar_id }}"
```

Comment:
This could just be called "journal_test_id".

Reply:
Also, I'm not 100% sure if this works for reporting. I need to check.

Reply:
It does work! https://github.com/infrawatch/feature-verification-tests/pull/125
I'll add it into the custom_logger tests since it's going to be a behaviour we rely on

Reply:
fixed name.